idf_component.yml

```diff
@@ -0,0 +1,4 @@
+description: LoRaWAN board port for ESP32
```

Comment:
can we make this more clear? this is loramacnode port for esp32 no?

Reply:
done